client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -8,11 +8,14 @@
 </license>
 
 <template>
-  <div>
+  <div
+    :class="{ 'top-0 left-0 w-full h-full fixed z-[2200] bg-white': isFullscreen }">
     <dp-sticky-element
       border
-      class="u-pv-0_5">
-      <div class="flex items-start space-inline-s">
+      class="py-4"
+      :class="{ 'fixed top-0 left-0 w-full h-[15%] px-4': isFullscreen }">
```

Comment:
Why do you have to set the height here? I'd expext the element takes the height defined by its content.

Reply:
you are right, it can be removed as its height is defined by the content. done here https://github.com/demos-europe/demosplan-core/pull/2979/commits/a842dfe8552b4c374486dcc41c97d3f971f4e2b2